Documentation/help_files/openemr_installation_help.php

```diff
@@ -0,0 +1,589 @@
+                    
+                    <p><?php echo xlt("The Zend Framework is a collection of PHP packages used in OpenEMR to make the application work");?>.
+                    
+                    <p><?php echo xlt("The location of the Apache configuration file is dependent on the operating system");?>.
+                    
+                    <p><?php echo xlt("In windows, the XAMPP 1.7.0 package places the configuration file at xampp\apache\conf\httpd.conf");?>.
+                    
+                    <p><?php echo xlt("On most Linux systems if you installed Apache with a package manager, or it came pre-installed, the Apache configuration file is located in one of these locations");?>:
+                        <ul>
+                            <li><?php echo xlt("/etc/apache2/httpd.conf"); ?></li>
+                            <li><?php echo xlt("/etc/apache2/apache2.conf"); ?></li>
+                            <li><?php echo xlt("/etc/httpd/httpd.conf"); ?></li>
+                            <li><?php echo xlt("/etc/httpd/conf/httpd.conf"); ?></li>
+                        </ul>
+                    
+                    <p><?php echo xlt("On macOS Server >= 10.8 Mountain Lion, the location of the Apache configuration file varies");?>.
+                    
+                    <p><?php echo xlt("Not using websites /webservices	- default (/etc/apache2/httpd.conf)");?>.
+                    
+                    <p><?php echo xlt("macOS Server 4 - webservices - /Library/Server/Web/Config/apache2/httpd_server_app.conf");?>.
+                    
+                    <p><?php echo xlt("macOS Server 5 websites -	/Library/Server/Web/Config/apache2/server-httpd.conf");?>.
+                    
+                    <p><?php echo xlt("To configure Zend and to secure the /documents, /edi and /era directories you can paste following to the end of the apache configuration file");?>:
+                    
+                    <pre>
+                    &lt;Directory &quot;openemrwebroot&quot;&gt;
+                    &nbsp;&nbsp;AllowOverride FileInfo
+                    &lt;/Directory&gt;
+                    &lt;Directory &quot;openemrwebroot/sites&quot;&gt;
+                    &nbsp;&nbsp;AllowOverride None
+                    &lt;/Directory&gt;
+                    &lt;Directory &quot;openemrwebroot/sites/*/documents&quot;&gt;
+                    &nbsp;&nbsp;order deny,allow
+                    &nbsp;&nbsp;Deny from all
+                    &lt;/Directory&gt;
+                    &lt;Directory &quot;openemrwebroot/sites/*/edi&quot;&gt;
+                    &nbsp;&nbsp;order deny,allow
+                    &nbsp;&nbsp;Deny from all
+                    &lt;/Directory&gt;
+                    &lt;Directory &quot;openemrwebroot/sites/*/era&quot;&gt;
+                    &nbsp;&nbsp;order deny,allow
+                    &nbsp;&nbsp;Deny from all
+                    &lt;/Directory&gt;
+                    </pre>
```

Comment:
Recommend customizing the paths above (if possible), like is done in the setup.php script.

Reply:
Actually, after further thought, probably not a good idea to do this in a script that is not password protected (ie. should not let it be known specifically where things are by just looking at this unprotected script). Recommend instead change path to a placeholder like \<webroot\>